india_compliance/income_tax_india/overrides/test_asset_depreciation_schedule.py

```
@@ -0,0 +1,426 @@
+        asset.submit()
+
+        schedule = get_asset_depr_schedule_doc(asset.name, "Active", self.fb_it.name)
+        self.assertTrue(schedule.flags.get("wdv_it_act_applied"))
+
+        fy_start = "2024-04-01"
+        fy_end = get_fiscal_year(fy_start)[2]
+        total = flt(
+            sum(
+                row.depreciation_amount
+                for row in schedule.depreciation_schedule
+                if row.schedule_date <= getdate(fy_end)
```

Flake8 (7.3.0)

[error] undefined name 'getdate'

(F821)